### PR TITLE
Docs for UXP v7.0

### PR DESCRIPTION
**Pull request by padmkris123**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/uxp-api/reference-js/Global Members/HTML DOM/AbortController.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
```
Comment (padmkris123) [inline src/pages/uxp-api/reference-js/Global Members/HTML DOM/AbortController.md:1]: @kareemmyk  I dont see the reference of AbortController and AbortSignal in Changelog. Is this intended for UXPv7.0?

File: src/pages/uxp-api/reference-js/Global Members/HTML DOM/AbortSignal.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+
```
Comment (padmkris123) [inline src/pages/uxp-api/reference-js/Global Members/HTML DOM/AbortSignal.md:1]: @kareemmyk  I dont see the reference of AbortController and AbortSignal in Changelog. Is this intended for UXPv7.0?

File: src/pages/uxp-api/reference-js/Modules/uxp/Persistent File Storage/errors.md [hunk 3]
```diff
@@ -109,3 +106,11 @@ Unsupported format type
 Data and Format mismatch
 
 
+
+<a name="module-storage-errors-notsupported" id="module-storage-errors-notsupported"></a>
+
+## NotSupported
+Not supported error
```
Comment (padmkris123) [inline src/pages/uxp-api/reference-js/Modules/uxp/Persistent File Storage/errors.md:113]: @kareemmyk  We had earlier decided not to show this. Is this needed now?

File: src/pages/uxp-api/reference-js/Modules/fs/fs.md [hunk 4]
```diff
@@ -1,29 +1,30 @@
...
 UXP supports plugin-specific storage schemes, such as "plugin:", "plugin-data:",
-and "plugin-temp:", as well as a native "file:" scheme for the path parameter.<br/>
-Note1: [UWP](https://learn.microsoft.com/en-us/windows/uwp/get-started/universal-application-platform-guide)(Universal Windows Platform)
+and "plugin-temp:", as well as a native "file:" scheme for the path parameter.<br>
+<b>Note:</b><br></br>
+1. If there are no schemes defined for the path parameter of FSAPI methods, it considers to have "file:" scheme for the path.<br></br>
+2. [UWP](https://learn.microsoft.com/en-us/windows/uwp/get-started/universal-application-platform-guide)(Universal Windows Platform)
 has the strict [File access permissions](https://learn.microsoft.com/en-us/windows/uwp/files/file-access-permissions),
-and UXP FSAPI may have access issues with anonymous filepaths.
-So, XD does not support this feature for compatibility across platforms.<br/>
```
Comment (padmkris123) [inline src/pages/uxp-api/reference-js/Modules/fs/fs.md:14]: @kareemmyk  These should be called out in 'Known Issues' section.

**Comment by padmkris123** [comment]
@kareemmyk  I didn't see any changes to `reference-css`. Could you confirm if thats correct? Hopefully, I haven't missed anything.

Also, I havent taken anything from storage.md - please confirm that I dont need to check it. Lets remove it since all the changes should go into the respective API files.

File: src/pages/uxp-api/reference-js/Modules/fs/fs.md (hunk 4 above)
Comment (kareemmyk) [reply src/pages/uxp-api/reference-js/Modules/fs/fs.md:14]: Updated in the branch ```v7.0.0-docs-review```

File: src/pages/uxp-api/reference-js/Modules/uxp/Persistent File Storage/errors.md (hunk 3 above)
Comment (kareemmyk) [reply src/pages/uxp-api/reference-js/Modules/uxp/Persistent File Storage/errors.md:113]: Added @ignore annotation. We can skip it

File: src/pages/uxp-api/reference-js/Global Members/HTML DOM/AbortSignal.md (hunk 2 above)
Comment (kareemmyk) [reply src/pages/uxp-api/reference-js/Global Members/HTML DOM/AbortSignal.md:1]: @padmkris123  .... The APIs for AbortController and AbortSignal was added long back.
But we added some documentation for those APIs in the current release.
Hence, it's not added into the changelog.

File: src/pages/uxp-api/reference-js/Global Members/HTML DOM/AbortController.md (hunk 1 above)
Comment (kareemmyk) [reply src/pages/uxp-api/reference-js/Global Members/HTML DOM/AbortController.md:1]: @padmkris123  .... The APIs for AbortController and AbortSignal was added long back.
But we added some documentation for those APIs in the current release.
Hence, it's not added into the changelog.

File: src/pages/uxp-api/reference-js/Global Members/HTML DOM/Alerts/confirm.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+
+<a name="confirm" id="confirm"></a>
```
Comment (padmkris123) [inline src/pages/uxp-api/reference-js/Global Members/HTML DOM/Alerts/confirm.md:2]: @kareemmyk  Possible to move alert.md, confirm and prompt within an 'Alerts' folder in torq-native?

File: src/pages/uxp-api/reference-js/Global Members/ImageBlob/index.md [hunk 6]
```diff
@@ -1,2 +1,189 @@
-# ImageBlob
-* [ImageBlob](ImageBlob.md)
+
```
Comment (padmkris123) [inline src/pages/uxp-api/reference-js/Global Members/ImageBlob/index.md:1]: @kareemmyk  Move the content of ImageBlob.md to index.md in torq-native

File: src/pages/uxp-api/reference-js/Modules/fs/index.md [hunk 7]
```diff
@@ -1,2 +1,401 @@
-# fs
-* [fs](fs.md)
+
```
Comment (padmkris123) [inline src/pages/uxp-api/reference-js/Modules/fs/index.md:1]: @kareemmyk  Please move the content from fs.md to index.md in torq-native

File: src/pages/uxp-api/reference-js/Global Members/Path/index.md [hunk 8]
```diff
@@ -1,2 +1,221 @@
-# Path
-* [Path](Path.md)
+
```
Comment (padmkris123) [inline src/pages/uxp-api/reference-js/Global Members/Path/index.md:1]: @kareemmyk  Please move contents from Path.md to index.md in torq-native

**Comment by kareemmyk** [comment]
> @kareemmyk I didn't see any changes to `reference-css`. Could you confirm if thats correct? Hopefully, I haven't missed anything.
> 
> Also, I havent taken anything from storage.md - please confirm that I dont need to check it. Lets remove it since all the changes should go into the respective API files.

Yes, there are no changes to ```reference-css```
Yes, no new changes added for storage.md

File: src/pages/uxp-api/reference-js/Global Members/HTML DOM/Alerts/confirm.md (hunk 5 above)
Comment (kareemmyk) [reply src/pages/uxp-api/reference-js/Global Members/HTML DOM/Alerts/confirm.md:2]: We did investigate this option.
This is a limitation of the source code for docs generation. We can have a parent and a child folder but no additional folder inside of it. In this case, it's Global Members(parent) -> HTML DOM(child).
So at this stage the current state is our best option

File: src/pages/uxp-api/reference-js/Global Members/ImageBlob/index.md (hunk 6 above)
Comment (kareemmyk) [reply src/pages/uxp-api/reference-js/Global Members/ImageBlob/index.md:1]: This is a limitation of the source code for docs generation. Once we group a file into a folder, then a master ```index.md``` get's created by default
So at this stage the current state is our best option

File: src/pages/uxp-api/reference-js/Modules/fs/index.md (hunk 7 above)
Comment (kareemmyk) [reply src/pages/uxp-api/reference-js/Modules/fs/index.md:1]: This is a limitation of the source code for docs generation. Once we group a file into a folder, then a master ```index.md``` get's created by default
So at this stage the current state is our best option

File: src/pages/uxp-api/reference-js/Global Members/Path/index.md (hunk 8 above)
Comment (kareemmyk) [reply src/pages/uxp-api/reference-js/Global Members/Path/index.md:1]: This is a limitation of the source code for docs generation. Once we group a file into a folder, then a master ```index.md``` get's created by default
So at this stage the current state is our best option

**Pull request merged**